.gitignore

```diff
@@ -1,3 +1,5 @@
 /storage.spec.whatwg.org/
 /deploy.sh
 /storage.html
+/node_modules/
+package-lock.json
```

Comment:
This we'll also have to change upstream: https://github.com/whatwg/spec-factory/blob/main/factory.json. But it's straightforward enough.

Reply:
Will it override this file if unchanged btw?